website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -163,6 +163,35 @@ If no filters have been applied, use a [Tooltip](/components/tooltip) coupled wi
 
 ![Empty state in applied filters](/assets/patterns/filter-patterns/applied-filters-empty-state.png =450x*)
 
+### Avoiding empty state
+
+Because filtering is often reductive in nature and driven by a user action, it might not always be possible to avoid an empty state within a data set. This can be helped by leveraging specific component variants and properties, and by taking steps to prevent mistakes or errors that would result in an empty state prior to them occurring.
+
+#### ListItem count
+
+Use the `count` property within the Dropdown ListItem to more clearly explicitly call attention to filters that will return zero results.
+
+In this example, `status = pending` does not correspond with any results.
+
+![Count property in ListItem](/assets/patterns/filter-patterns/avoid-empty-state-list-item-count.png)
+
+#### Approximate matching
+
+To help resolve basic spelling mistakes, typos, and broaden the relevancy of a search term being filtered upon, consider using _approximate_ matching or "fuzzy search" when filtering by a string value.
```

Comment:
Is there a good resource you could link to for folks who may not be as familiar with "fuzzy search"?